plugins/auth-backend/src/providers/types.ts

```diff
@@ -196,6 +196,10 @@ export type ProfileInfo = {
    * signed in user.
    */
   picture?: string;
+  /**
+   * Username of the signed in user.
+   */
+  username?: string;
```

Comment:
Let's skip this as part of the profile since the profile is only intended to be a representation of the logged in account for the end user. In this case we'd just use the `username` as `displayName` if a full name is unavailable.